### DISCUSSION TITLE
谈易语言 | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#297] [General]
# 谈易语言 | 愧怍

愧怍的个人博客

https://kuizuo.cn/blog/easy-language/

**Comment by Skywt2003** [comment]
我初中第一次接触编程也是易语言。后来学了 Visual Basic 才知道这玩意就是中文版的 Visual Basic……只能说是当时对编程语言的世界了解不足，才会选择这样一个语言入门。毕竟用中文编程还是有点逆天的……
如果有合适的领路人，或者当时能看到合适的书籍带我入门编程的世界，我觉得我肯定不会接触这个语言 =_=

**Reply by kuizuo** [reply]
无论第一门编程语言是什么，只要能够让自己有兴趣，肯为之学习下去，就是一门好语言。
现在回看易语言，设计的不是一般垃圾，没有 git 版本控制、调试无法更代码、时不时就程序崩溃。换现在如此注重开发体验的我，我是绝不会接触这门语言。
但不可否认的是，易语言是我的启蒙编程语言，可以说没有易语言就没有现在还在接触 IT 行业的我。
